client/src/utils/gen-util.js

```diff
@@ -18,11 +18,14 @@ export const sortObjects = (items, key, direction = 'asc') => {
     return [];
   }
   return items.sort((itema, itemb) => {
+    const a = itema[key] ?? '';
```

Comment:
```suggestion
    // guard against null values
    const a = itema[key] ?? '';
```

Reply:
I had one actually and removed it before I committed. I'll add one back

Reply:
Hmm, this comment is describing what the code is doing rather than explaining what we want the code to do
Typically we want the latter over the former